peachjam/templates/peachjam/court_detail.html

```diff
@@ -27,11 +27,16 @@ <h1 class="mt-4">{{ formatted_court_name }}</h1>
   {% include 'peachjam/_document_count.html' %}
   {% include 'peachjam/_registries.html' %}
   {% include 'peachjam/_years_list.html' with showallyears=True %}
+  {% if months %}
+    {% include 'peachjam/_months_list.html' with showallmonths=True %}
+  {% endif %}
 {% endblock %}
 {% block content %}
-  {% if documents %}
+  {% if documents and court_detail_view %}
```

Comment:
I'm pretty sure we always want to show a grouped judgments table, except when the list is sorted alphabetically and not by date, in which case grouping doesn't apply.